app/src/shared/components/Layout/layout.pcss

```diff
@@ -30,9 +30,7 @@
   .mobileNav {
     display: none;
   }
-}
```

Comment:
Nav was hidden on mobile and desktop and not on tablet. Seems like unintentional thing. Fixed it.